components/skeleton/skeleton-mixin.js

```diff
@@ -155,7 +156,8 @@ export const SkeletonMixin = dedupeMixin(superclass => class extends RtlMixin(su
 			 * Renders the input as a [skeleton loader](https://github.com/BrightspaceUI/core/tree/main/components/skeleton)
 			 * @type {boolean}
 			 */
-			skeleton: { reflect: true, type: Boolean  }
+			skeleton: { reflect: true, type: Boolean  },
+			skeletonDisplay: { reflect: true, type: Boolean, attribute: 'skeleton-display' },
```

Comment:
Or, maybe crazy idea... does this work, or totally break things?
```
skeleton: { type: Boolean, attribute: none },
_skeletonDisplay: { reflect: true, type: Boolean, attribute: 'skeleton' },
```
🤔 Like can we keep accept `skeleton` as the property, but then control the setting of the attribute? Lit might not like it, it might mean we'd need to update every consumer to target the property?

Reply:
That's an interesting idea!

Another crazy one that I haven't fully thought through: can we create a getter/setter for the `skeleton` property and use that? Like the getter would return what you have for `skeleton-display` and the setter would save the value for later.

Then we wouldn't need to update any consumers.